events/moose/moose.go

```diff
@@ -683,6 +683,18 @@ func (s *Subscriber) NotifyDebuggerEvent(e events.DebuggerEvent) error {
 	return s.response(moose.NordvpnappSendDebuggerLoggingLog(e.JsonData, combinedPaths, nil))
 }
 
+func (s *Subscriber) NotifyAppStartTime(duration int64) error {
+	if duration > math.MaxInt32 || duration < math.MinInt32 {
```

Comment:
Maybe we should simply clamp it to <0, MaxInt32>? Everything that's outside of this scope, produces here an error